ffi/capi/src/casemap.rs

```diff
@@ -115,6 +115,7 @@ pub mod ffi {
             hidden
         )]
         #[cfg(feature = "compiled_data")]
+        #[diplomat::attr(demo_gen, disable)]
```

Comment:
I won't leave comments everywhere, but please add comments explaining why you're disabling these. In this case I understand; these static functions duplicate functionality available on methods.